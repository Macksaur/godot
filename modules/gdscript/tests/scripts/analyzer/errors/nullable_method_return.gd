func test():
    print(_get_result())


func _get_result() -> String:
    var might_be_null: String? = null
    return might_be_null
