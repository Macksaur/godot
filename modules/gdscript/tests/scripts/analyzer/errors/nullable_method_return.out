GDTEST_ANALYZER_ERROR
Cannot return nullable value of type "String?" because the function return type is "String".
